easybuild/easyconfigs/f/FSL/FSL-6.0.2-foss-2018b-Python-2.7.15-CUDA-9.2.88.eb

```diff
@@ -0,0 +1,86 @@
+# This build still relies on the following libraries (or part of them) shipped with FSL:
+#    GDCHART, libprob, libcprob, cprob, newran, newmat
+# Attempts to use externally built libraries failed. Worth trying again in the future...
+#
+# Some tools like icmp are missing since they are built with the fslpython_install.sh script
+# which locally installs Miniconda and creates a virtual environment. This is too messy and
+# an alternative solution is necessary to install them.
+# https://github.com/easybuilders/easybuild-easyconfigs/issues/7899
+# https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FslInstallation#imcp.2Fimglob.2Fimmv_errors_after_installation
+
+name = 'FSL'
+version = '6.0.2'
+local_python_suffix = '-Python-2.7.15'
+local_CUDA_ver = '9.2.88'
+versionsuffix = '{0}-CUDA-{1}'.format(local_python_suffix, local_CUDA_ver)
+
+homepage = 'https://www.fmrib.ox.ac.uk/fsl/'
+description = """FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = ["https://www.fmrib.ox.ac.uk/fsldownloads/"]
+sources = ['%(namelower)s-%(version)s-sources.tar.gz']
+patches = [
+    'FSL-6.0.2_Makefile_fixes.patch',
+    'FSL-6.0.2_Build_extras.patch',
+    'FSL-6.0.1_Melodic-use-ifstream-good.patch',
+    'FSL-6.0.2_Enable_GPU_build.patch',
+    # Fixes https://www.jiscmail.ac.uk/cgi-bin/webadmin?A2=ind1911&L=FSL&P=R2783
+    'FSL-6.0.2_probtrackx2_gpu_large_mri_support.patch',
+    'FSL-6.0.1_Fix_fsl_exec_script.patch',
+]
+checksums = [
+    'c118b351c6cedb441af7e1b9d194cf344505ff53b417063f697b86305a908afd',  # fsl-6.0.2-sources.tar.gz
+    '3a14d9c3ea75d90fa3e4beb5da3cd11a60c6859441acaece02b32ccab6aa278f',  # FSL-6.0.2_Makefile_fixes.patch
+    '2b7ade2146fadbba6c983370bb030ac4e2b11d8bf5259eacc8f3054d3d11003f',  # FSL-6.0.2_Build_extras.patch
```

Comment:
@casparvl these SHA256 checksums are not correct, they should be updated

Reply:
Darn, you're right. I updated the patches with descriptions for the other (non-GPU) PR, but failed to update these. Doing so now...